packages/product-download-page/style.module.css

```diff
@@ -28,33 +29,29 @@
   margin: 40px auto 64px;
   text-align: center;
 
-  & p {
-    color: var(--gray-2);
-  }
-
-  & a {
-    color: var(--black);
-  }
-
-  & h2 {
-    text-align: center;
-    margin: 16px 0;
-  }
-
   @media (--medium-up) {
     margin: 88px auto 128px;
   }
 }
 
+.gettingStartedTitle {
+  composes: g-type-display-2 from global;
+  text-align: center;
+  margin: 16px 0;
+}
+
+.gettingStartedDescription {
+  composes: g-type-body from global;
+  color: var(--gray-2);
+}
+
 .links {
   margin-top: 32px;
-  & a {
-    text-align: center;
-    display: block;
-    margin: 8px 0;
-  }
 }
 
-.signUpButton {
```

Comment:
The `signUpButton` class was unused, so I cut it. Also split out the `.link` class for clarity.